properties/properties.go

```diff
@@ -60,11 +60,14 @@ func (qis *QuiteliteInputSource) getVal(name string) (interface{}, bool) {
 func (qis *QuiteliteInputSource) Int(name string) (int, error) {
 	otherGenericValue, exists := qis.getVal(name)
 	if exists {
-		otherValue, isType := otherGenericValue.(int)
+		otherValue, isType := otherGenericValue.(float64)
```

Comment:
Does quitelite treat numbers, including integer, as float64?  Can its implementation change in the future that returns int for integer value?

Reply:
Ye
Yes, see here: https://golang.org/pkg/encoding/json/

> To unmarshal JSON into an interface value, Unmarshal stores one of these in the interface value:

```
bool, for JSON booleans
float64, for JSON numbers
string, for JSON strings
[]interface{}, for JSON arrays
map[string]interface{}, for JSON objects
nil for JSON null
```